src/Parsers/XML.php

```diff
@@ -5,6 +5,8 @@
 
 class XML
 {
+    const REGEX_ASCII_CONTROL_CHARACTERS = '/(&#[xX](0?[0-9]|[12][0-9]|3[01]);)/'; // '&#xN;' where 0 <= N < 32
```

Comment:
since the composer.json only specifies `"php": ">=5.5.0"`, PhpStorm rightfully didn't like it when I had this as `private const`